en/flight_modes/manual_fw.md

```diff
@@ -0,0 +1,38 @@
+# Manual Flight Mode (Fixed Wing)
+
+<!-- this requires review and updates -->
+
+[<img src="../../assets/site/difficulty_6.svg" title="Difficulty (Hardest)" width="30px" />](../getting_started/flight_modes.md#key_difficulty)&nbsp;[<img src="../../assets/site/remote_control.svg" title="Manual/Remote control required" width="30px" />](../getting_started/flight_modes.md#key_manual)&nbsp;
+
+*Manual mode* sends RC stick input directly to the output mixer for fully manual control.
+
+> **Tip** This is the hardest mode to fly, because nothing is stabilized. Unlike [Acro Mode](../flight_modes/acro_fw.md) if the RP stick is centered the vehicle will not automatically stop rotating around the axis; the pilot actually has to move the stick to apply force in the other direction. 
+
+<span></span>
+> **Note** This is the only mode that overrides the FMU (commands are sent via the safety coprocessor). It provides a safety mechanism that allows full control of throttle, elevator, ailerons and rudder via RC in the event of an FMU firmware malfunction.
+
+## Technical Description
+
+RC mode where stick input is sent directly to the output mixer (for "fully" manual control).
+
+This is the only mode that overrides the FMU (commands are sent via the safety coprocessor). It provides a safety mechanism that allows full control of throttle, elevator, ailerons and rudder via RC in the event of an FMU firmware malfunction.
+
+## Parameters
+
+Parameter | Description
+--- | ---
+&nbsp; | 
+
+<!-- Possibly of interest:

```

Comment:
Below are possibly relevant parameters. Should these be made visible? Are there any others?

Reply:
I think FW_MAN_P_MAX and FW_MAN_R_MAX don't apply for fully manual mode. The others are relevant, but I'm not sure if there are more.